<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ihm</class>
 <widget class="QWidget" name="Ihm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DARTS</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ecranAttente
{
		background-image:url(:/ressources/background1.png)
	/*background-color: #FFFFFF; */
}
QWidget#ecranSeance
{
	background-color: rgb(35, 93,166);
}
QWidget#ecranResultats
{
	 background-color: rgb(35, 93,166);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="ecranDarts">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="ecranAttente">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutHeure">
           <item>
            <widget class="QLabel" name="labelHeureAttente">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>00:00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutLogo">
           <item>
            <widget class="QLabel" name="labelLogo">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="ressources.qrc">:/ressources/logoDarts.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutStatut">
           <item>
            <widget class="QLabel" name="labelStatutAttente">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>En attente de connexion</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ecranPartie">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutVisualisation">
           <item>
            <widget class="QLabel" name="labelHeurePartie">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelVisualisationimpact">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="ressources.qrc">:/ressources/cible.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelStatut">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutSeparateur">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutInformation">
           <item>
            <layout class="QFormLayout" name="formLayoutInformation">
             <item row="0" column="0">
              <widget class="QLabel" name="labelTypeJeu">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>- Type de jeu :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="typeJeu">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                </font>
               </property>
               <property name="text">
                <string>501</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="Line" name="lineSeparateurTypeJeu">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>5</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLabel" name="labelNomjoueurs">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>- Nom des joueurs :</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="Line" name="lineNomJoueurs">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labelManche">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>- Manche : </string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                </font>
               </property>
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="Line" name="lineManche">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <widget class="Line" name="lineScoreActuel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <widget class="QLabel" name="labelScoreActuel">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>- Score actuel : </string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>30</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>- Moyenne des vollées :</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ecranStatistique"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
